src/import/kv_importer.rs

```diff
@@ -59,6 +59,13 @@ impl KVImporter {
             return Ok(());
         }
 
+        // Restrict opening engine num
+        if inner.engines.len() >= self.cfg.num_engines {
+            let errmsg = format!("Too much opening engines {}: {}", uuid, inner.engines.len());
+            error!("{}", errmsg);
+            return Err(Error::RocksDB(errmsg));
```

Comment:
This is not a RocksDB error, please define a more specific error.

Reply:
Sure